common/containers/Tabs/Contracts/components/Interact/components/InteractExplorer/index.tsx

```diff
@@ -160,7 +186,7 @@ class InteractExplorerClass extends Component<Props, State> {
       const callData = { to: to.raw, data };
       const results = await nodeLib.sendCallRequest(callData);
 
-      const parsedResult = selectedFunction.decodeOutput(results);
+      const parsedResult = selectedFunction!.contract.decodeOutput(results);
```

Comment:
What is this syntax? Is this some typescript thing I don't know? AFAICT this is invalid vanilla javascript.

Reply:
Holy shit typescript has `?` from coffeescript, this is the best day of my life: https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

Reply:
Yeah, basically telling typescript that this won't be null at runtime.

Reply:
@wbobeirne This isnt the exact operator you're looking for. What you want is the `elvis` operator, which is a runtime operator. The null-assertion operator overrides typescript saying that the value will not be null at run time